spec/unit/gettext_config_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'puppet/gettext/config'
+require 'spec_helper'
+
+describe Puppet::GettextConfig do
+  require 'puppet_spec/files'
+  include PuppetSpec::Files
+
+  let(:local_path) do
+    local_path ||= Puppet::GettextConfig::LOCAL_PATH
```

Comment:
Actually, I have a question about those constants I've defined. Is the code used to set them (which checks on the filesystem) evaluated every time they're used, or just once when the file is loaded? If the latter, this is clearly redundant. All the examples I'm seeing for constants don't do any work (e.g. they're just numbers), so the question isn't really relevant...